opal/mca/common/ofi/common_ofi.c

```diff
@@ -685,3 +686,72 @@ struct fi_info *opal_mca_common_ofi_select_provider(struct fi_info *provider_lis
     free(provider_table);
     return provider;
 }
+
+/**
+ * Obtain EP endpoint name
+ *
+ * This function obtained the EP endpoint name and length for the
```

Comment:
It is not good style to start comments with the function name or this function. The obtained makes wonder whether I should call the function all. It seems to be finished already. I would prefer obtains.